### PR TITLE
CPDP-514: adhere to enrolment rules in service config

### PR DESCRIPTION
**Pull request by co-category**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposals/service/TaxEnrolmentService.scala [hunk 1]
```diff
@@ -60,7 +60,7 @@ class TaxEnrolmentServiceImpl @Inject()(connector: TaxEnrolmentConnector) extend
         maybePostcode match {
           case Some(postcode) =>
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposals/service/TaxEnrolmentService.scala]: should we be sending the country code here for a non-uk address?

Comment (co-category) [reply app/uk/gov/hmrc/cgtpropertydisposals/service/TaxEnrolmentService.scala]: I'm not sure of the requirements here. I have given postcode precedence for both scenarios. Should it be country code for Non-UK addresses?

Comment (co-category) [reply app/uk/gov/hmrc/cgtpropertydisposals/service/TaxEnrolmentService.scala]: changed to always use country-code for non-uk addresses

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**